SwiftGit2.xcodeproj/project.pbxproj

```diff
@@ -729,7 +729,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "if which swiftlint >/dev/null; then\n    swiftlint --lenient\nelse\n    echo \"warning: SwiftLint is not installed\"\nfi";
+			shellScript = "if which swiftlint >/dev/null; then\n    swiftlint\nelse\n    echo \"warning: SwiftLint is not installed\"\nfi";
```

Comment:
This is giving me the error like unknown flag when I tried to compiled the code, not sure what happened there. I've tried googling also, nothing come up. Any suggestions? My SwiftLint version is wrong?